packages/boiler-task-selenium/src/runner/spawn.js

```diff
@@ -0,0 +1,18 @@
+// Helpers
+import runWebdriverio from './webdriverio/run';
+import runNightwatch from './nightwatch/run';
+
+
+/**
+ * Given a test configuration object, spawn a test runner instance
+ *
+ * @param {Array} capabilities
+ * @param {Object} runnerOptions
+ * @param {Object} config // Task configuration
+ * @param {Boolean} forceTunnel
+ * @param {Function} cb
+ */
+export default function spawn(capabilities, runnerOptions, config, forceTunnel, cb) {
+  const run = runnerOptions.runner === 'nightwatch' ? runNightwatch : runWebdriverio;
+  run(capabilities, runnerOptions, config, forceTunnel, cb);
+}
```

Comment:
if you wanna be a hipster you can do `run.apply(null, arguments` or in es6 I think `run(...args)` works if you use `...args` in your wrapper function rather than declaring all the arguments.